lib/hawkular/hawkular_client.rb

```diff
@@ -18,6 +16,10 @@ def initialize(hash)
       @state = hash
     end
 
+    def respond_to_missing?
+      true
```

Comment:
I think this is wrong. It should only return `true` if `method_missing` will process the call (won't fail).